common/council_tasks/00_steward_tasks.txt

```diff
@@ -584,9 +584,8 @@ task_promote_culture = {
 				desc = STEWARD_PROMOTE_CULTURE_COMMUNAL_IDENTITY_BONUS_MODIFIER
```

Comment:
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/113168881-1e696b00-9256-11eb-992d-111ed64984ce.png)

</details>